### ISSUE TITLE
[MSFT-227] "FORUM POSTS" link foreground and background colors missed WCAG 2 AA contrast ratio

### ISSUE THREAD

**Issue by veshu** [importtest/demo#25]
**Steps to Reproduce**
1. Go to: https://www.topcoder.com/members/doremihong
2. Right click > Inspect Elements 
3. Check the "FORUM POSTS" link

**Expected Result**
- Contrast ratio should be 4.5:1
- If you want passes WCAG 2 AA, you can change text color to #757588. New contrast ratio: 4.5:1

**Actual Result**
Foreground Color: #55A5FF
Background Color: #FFFFFF
Contrast Ratio: 2.55:1
=> Therefor, "FORUM POSTS" link foreground and background colors missed WCAG 2 AA contrast ratio

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome
**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F28
